File: c/3-loops/practice/lp-2.c
#include <stdio.h>
int main()
{
    int a;
    scanf("%d\n",&a);
    if(a<0||a>10000)
    {
        printf("Invalid input");
    }
    else{
        for(int i=1;i<=a;i++)
    {
        if(a%i==0)
        {
            printf("%d ",i);
        }
    }
    
    }
}

File: python/7-lists/practice/lp-2.py
a = list(map(int, input().split()))
if any(x < 0 for x in a):
    print("Invalid input")
else:
    even = sum(1 for x in a if x % 2 == 0)
    odd = sum(1 for x in a if x % 2 != 0)
    print(even)
    print(odd)